.github/workflows/scanner-build.yaml

```diff
@@ -149,3 +149,12 @@ jobs:
         architectures="amd64,arm64,ppc64le,s390x"
 
         push_scanner_image_manifest_lists "$architectures"
+
+  run-e2e-tests:
```

Comment:
If possible, we should move this into a different file. I don't think we should say the build step failed when/if the test fails

Reply:
Maybe @BradLugo would know how to do this, if possible?

Reply:
going to resolve, as this only blocks the build when the label is added